go/stellar/global.go

```diff
@@ -59,3 +61,8 @@ func (s *Stellar) GetServerDefinitions(ctx context.Context) (ret stellar1.Stella
 
 	return s.cachedServerConf, nil
 }
+
+// Pull the remoter off the global G's Stellar object
+func GetRemoter(g *libkb.GlobalContext) remote.Remoter {
+	return g.GetStellar().(*Stellar).Remoter
```

Comment:
If G is not properly initialized such `G.stellar.Remoter` is specifically a `remote.Remoter` then this will panic with `panic: interface conversion`. I'm ok with this.